packages/gafl-webapp-service/src/handlers/__tests__/renewal-start-date-validation-handler.spec.js

```diff
@@ -0,0 +1,57 @@
+import renewalStartDateValidationHandler from '../renewal-start-date-validation-handler'
+import { LICENCE_SUMMARY, RENEWAL_START_DATE } from '../../uri.js'
+
+describe('Renewal start date validation handler', () => {
+  const getMockRequest = (startYear, startMonth, startDay, renewedEndDate) => ({
+    cache: () => ({
+      initialize: () => ({}),
+      helpers: {
+        page: {
+          getCurrentPermission: async () => ({
+            payload: {
+              'licence-start-date-year': startYear || 1970,
```

Comment:
Rather than using this construct, I'd suggest using default arguments would be a bit easier to read:
```
const getMockRequest = (startYear = 1970, startMonth = 1, startDat = 1, renewedEndDate = '2023-01-01T00:00:00.000Z') => ({
```

Reply:
Good call – will sort this out tomorrow.